tools/report-converter/tests/unit/mdlint_output_test_files/Makefile

```diff
@@ -0,0 +1,2 @@
+all:
+	mdl ./files/readme.md > readme.out
```

Comment:
maybe it is worth to mention here or somewhere in the test documentation that there is a docker file which can be used to install and execute markdownlint easier:
```
docker run --rm -v ${PWD}:/data markdownlint/markdownlint .
```